vocab_list/models.py

```diff
@@ -0,0 +1,55 @@
+from django.conf import settings
+from django.db import models
+
+
+class VocabularyList(models.Model):
+
+    # a NULL owner means a system-provided vocabulary list
+    owner = models.ForeignKey(
+        getattr(settings, "AUTH_USER_MODEL", "auth.User"),
+        null=True,
+        blank=True,
+        on_delete=models.CASCADE
+    )
+    title = models.CharField(max_length=255)
+    description = models.TextField(blank=True)
+
+    created = models.DateTimeField(auto_now_add=True)
```

Comment:
There has been talk for a while about deprecating `auto_now_add` and `auto_now`.  I'm +0 for using `default=timezone.now` instead and then having:

```python
def save(self, *args, **kwargs):
    self.updated = timezone.now()
    return super().save(*args, **kwargs)
```

Could put these two fields and the save method on a base model class.

Reply:
Well, it was frowned upon years ago but looking at the Django 2.1 docs, it seems it's back in favour.

James Harrington's original code had abstract base classes but I felt it overcomplicated things at this stage.